internal/runbits/buildscript/commitid.go

```diff
@@ -0,0 +1,36 @@
+package buildscript_runbit
+
+import (
+	"errors"
+
+	"github.com/go-openapi/strfmt"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/pkg/buildscript"
+	"github.com/ActiveState/cli/pkg/checkoutinfo"
+)
+
+func CommitID(path string, cfg configurer) (strfmt.UUID, error) {
```

Comment:
Can we move this to checkoutinfo? I get where you're coming from, but runbits are about sharing controller logic. This isn't controller logic. And this type of muliplexing is exactly what the checkoutinfo package is intended for.